### ISSUE TITLE
Game engine setup explaination

### ISSUE THREAD

**Issue by ExcaliburZero** [Infinite-Dango-Studios/ganbatte#6]
I just want to go over how the setup with the game engine we are using will work. Feel free to ask any questions you have, as it may be a bit confusing.

The game engine we are going to be using is called Ren'Py. It is a visual novel game engine, which uses a modified version of Python as its coding language.

Each member of the team will need to download a copy of Ren'Py (http://www.renpy.org/). Ren'Py will be able to run the source files provided in the repository, and thus be able to run the game.

Now, for keeping each of our local copies of the game up to date on the source files, there are two options we can choose from.
1. We can all download GitHub for Windows, which will allow each of us to create a clone (copy) of the repository (project) and then there should be a simple function in the program to update the clone.
2. We could each download the ".zip" file contain the contents of the repository, unzip it, and place it in the correct location, every time we need to update out local version of the project.

Please let me know what you think about this, and let me know what questions you have.

**Comment by juanitaylor** [comment]
wah! this is all technical! I don't fare too well with tech-y stuff XD I think the first thing might work better for me?

**Comment by ExcaliburZero** [comment]
@juanitaylor 
Yeah, I wasn't sure which one to go with.

Way number one requires the GitHub program, and thus requires a little bit more knowledge, but once the process is ironed out, it should be simple and quick. While way number two does not require as much technical knowledge, but is more complicated and repetitive.

**Comment by juanitaylor** [comment]
that's what I was thinking. If we figure it out in the beginning, it will be easier in the long haul XD

**Comment by ExcaliburZero** [comment]
@juanitaylor 
Yep. I recommend you download the Ren'Py program when you are able to, and I think I will work on figuring out the GitHub setup soon.

**Comment by ExcaliburZero** [comment]
I think I've established most of the basics for the using of the GitHub for Windows program.

Once you download the program and input your login information, do the following:
- Open your internet browser and go to https://github.com/ExcaliburZero/ganbatte
- Click on the button in the right side bar titled "Clone in Desktop"
- Select GitHub
- Follow the instructions and select a folder to put the repository in

That should be the basics of it. It shouldn't be too difficult to figure out how to update the repository when there are changes, and unfortunately I can't really test it as I am the repository owner. Just let me know if you need any help.

**Comment by ExcaliburZero** [comment]
Who has gotten the setup done on their end so far?

If you're having any trouble getting it set up, let me know and I'll help.

**Comment by ExcaliburZero** [comment]
I know Sabre-Hecate has gotten his copy of Ganbatte working, but has anyone been able to setup and get Ganbatte running?

@Infinite-Dango-Studios/artists

**Comment by ExcaliburZero** [comment]
Has anyone else been able to get the setup working?
